File: src/AoC16/s16e01-python/s16e01/part1.py
"""Part 1 solution runner."""

import sys

from s16e01 import solve_part1


def main():
    """Read input from stdin and print part 1 result."""
    input_text = sys.stdin.read()
    result = solve_part1(input_text)
    print(result)


if __name__ == "__main__":
    main()

File: src/AoC16/s16e01-typescript/part1.ts
#!/usr/bin/env ts-node
/** Advent of Code - Part 1 */

import * as fs from "fs";
import { solvePart1 } from "./common";

// Read from stdin
const STDIN_FD = 0;
const data = fs.readFileSync(STDIN_FD, "utf-8").trim();
const result = solvePart1(data);
console.log(result);
